posthog/models/prompt.py

```diff
@@ -0,0 +1,159 @@
+from typing import Any, Dict, List
+
+from django.db import models
+from django.utils import timezone
+
+
+class PromptSequenceState(models.Model):
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["team", "person", "key"], name="unique sequence key for person for team")
+        ]
+
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    person: models.ForeignKey = models.ForeignKey("Person", on_delete=models.CASCADE)
+    key: models.CharField = models.CharField(max_length=400)
+
+    last_updated_at: models.DateTimeField = models.DateTimeField(default=timezone.now)
+    step: models.IntegerField = models.IntegerField(default=1)
+    completed: models.BooleanField = models.BooleanField(default=False)
+    dismissed: models.BooleanField = models.BooleanField(default=False)
+
+
+# this model is currently unused as we are running an experimentation config, might be used in the future
+# or completely removed if we allow feature flags to carry a payload, which would be a prompts configuration JSON
+class PromptSequence(models.Model):
```

Comment:
We can't delete models/fields for various reasons. If we're not using this we should remove it and only add it when we need it. Otherwise there's a risk it lives on in the code unused for a long time